scripts/us_census/acs5yr/subject_tables/s1201/README.md

```diff
@@ -0,0 +1,11 @@
+# Artefacts for US census ACS5YR S1201 (Marital Status) Subject Table import #
+
+This import includes the population characteristics (such as gender, age, race and labor force participation) characterized by the marital status.
+
+Years: 2010-2019
+Geo : Country, State, County and Place
+
+Important Notes :
+1. Data for 2010-2014 contain ‘NATIVITY’ label. However, it is not present in data for 2015-2019. 
+2. Duplicate column names were found in datasets for years 2010-2012.
+3. Local import scripts s1201/process.py and s1201/generate_col_map.py are created to handle duplicate column names seperately while loading the dataset.
```

Comment:
Can you please add a comment in the s1201/generate_col_map.py to indicate which parts differ from the common/ files?

Reply:
Done